js/ui/number_box/number_box.base.js

```diff
@@ -144,6 +144,17 @@ var NumberBoxBase = TextEditor.inherit({
         });
     },
 
+    _isMobileSupportInputMode: function() {
```

Comment:
```suggestion
    _isDeviceSupportInputMode: function() {
```

Reply:
This method also checks if mobile device is used. Desktop supports `inputmode` but this method return `false` in this case.